.github/workflows/test.yml

```diff
@@ -25,6 +25,6 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm install
-    - run: npm run full-test
+    - run: npm run test
```

Comment:
?

Reply:
> EDIT: oh spawnSync /bin/sh ENOENT
> what fun

Reply:
basically, im fine with these tests not running through CI due to not being able to download the data files. we could probably force git to do it through a submodule init or possibly by including it in the run instructions here, but its not important. having tests which you can run  at all is better than the status quo of them not existing

Reply:
Is there not an easy way to e.g. switch to `/bin/bash`?

Reply:
lmao you think bash is available but not sh?

Reply:
In hindsight yeah that's probably unlikely.

I wonder how they get away with not providing `sh`, though. I bet that breaks a lot of CI. I wouldn't be surprised if it's possible to enable somewhere in the GitHub Actions config.